router/src/server.rs

```diff
@@ -673,7 +674,7 @@ async fn completions(
                 seed,
                 top_n_tokens: None,
                 grammar: None,
-                ..Default::default()
+                adapter_id: model.as_ref().filter(|m| *m != "tgi").map(String::from),
```

Comment:
Can we remove this special value ? It should exist in the first place.

if openai *requires* a model id, then we should actually match it against the actual deployed model id, no ?